frappe/desk/query_report.py

```diff
@@ -780,3 +781,20 @@ def get_user_match_filters(doctypes, user):
 			match_filters[dt] = filter_list
 
 	return match_filters
+
+def validate_filters_permissions(report_name, filters=None, user=None):
+    if isinstance(filters, str):
+        filters = json.loads(filters)
+
+    if filters:
```

Comment:
```python
if not filters:
    return
```

Can just have this as the first line of the function, then the rest can be indented one level less.

Reply:
Additionally, do setup and run [pre-commit](https://pre-commit.com/)

Reply:
Okay